OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs

```diff
@@ -228,7 +228,7 @@ public virtual void PlayLaunchSounds()
 
 			var toPlayer = isAllied ? localPlayer ?? Self.Owner : localPlayer;
 			var speech = isAllied ? Info.LaunchSpeechNotification : Info.IncomingSpeechNotification;
-			Game.Sound.PlayNotification(Self.World.Map.Rules, toPlayer, "Speech", speech, toPlayer.Faction.InternalName);
+			Game.Sound.PlayNotification(Self.World.Map.Rules, toPlayer, "Speech", speech, toPlayer?.Faction.InternalName);
```

Comment:
This would address the conserns

```suggestion
			Game.Sound.PlayNotification(Self.World.Map.Rules, toPlayer, "Speech", speech, toPlayer?.Faction.InternalName ?? Self.Owner.Faction.InstanceName);
```

Reply:
Thanks.